styling/utils.js

```diff
@@ -0,0 +1,20 @@
+import { css } from 'styled-components'
+
+// eslint-disable-next-line import/prefer-default-export
+export const wordBreak = css`
+  // Add line breaks 
+  // Source: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/
+
+  /* These are technically the same, but use both */
```

Comment:
Why do we use both?

Reply:
This is the explanation from the website:

Reply:
I see. How about simply extending the comment to read `..., but use both for broader browser support`? That would bring the point across without having to read the article.